From c93b3c56725efc80b51d628fe3446eeeac970227 Mon Sep 17 00:00:00 2001
From: Alex Stewart <astewart.49c6@gmail.com>
Date: Mon, 8 Apr 2024 16:16:08 -0400
Subject: [PATCH] Revert "opkg-feed: Fix adding feeds with same name as
 architecture"

The new regexes added by the target commit invalidate the regex group
indexes used to extract the feed source information. As a result,
`opkg-feed list` operations fail to find feeds, in all cases.

Revert this change until a fix can be developed.

This reverts commit 67994e62dc598282830385da75ba9b1abbbda941.

Signed-off-by: Alex Stewart <astewart.49c6@gmail.com>

Upstream-Status: Inappropriate [upstream ticket]
---
 opkg-feed | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/opkg-feed b/opkg-feed
index 36d4463..25ef879 100755
--- a/opkg-feed
+++ b/opkg-feed
@@ -171,7 +171,7 @@ createFeedLineRegex()
 	#	1 = full source type with any quotes
 	#	2 = source type without quotes if quoted
 	#	3 = source type if unquoted
-	sourceTypePattern='("([^"](src|dist)(/gz)?)"|(src|dist)(/gz)?)\s+';
+	sourceTypePattern='("([^"]*)"|(\S+))\s+';
 
 	# Feed name capture groups (4, 5, 6)
 	#	4 = full feed name with any quotes
